apps/dialtone-documentation/docs/.vuepress/views/IconCatalog.vue

```diff
@@ -186,14 +197,47 @@ const selectIcon = (icon) => {
   else isPopoverOpen.value[icon.name] = !isPopoverOpen.value[icon.name];
 };
 
+const scrollToIcon = (iconName) => {
+  if (isMobile.value) {
+    isModalOpen.value = true;
+  } else {
+    isPopoverOpen.value[iconName] = !isPopoverOpen.value[iconName];
+    nextTick(() => {
+      const iconElement = document.getElementById(`in-${iconName}`); // Scroll to the opened popover
+      if (iconElement) {
+        // there is a style in dialtone-docs.less setting scroll-behavior: smooth, so
+        // we need to first set it to auto to avoid having the scroll animation
+        document.documentElement.style.scrollBehavior = 'auto';
+        iconElement.scrollIntoView({ behavior: 'instant' });
+        // set the scroll behavior to smooth again
+        // eslint-disable-next-line no-return-assign
+        setTimeout(() => document.documentElement.style.scrollBehavior = 'smooth', 5);
+      }
+    });
+  }
+};
+
 watch(selectedCategory, (newCategory) => {
   if (newCategory === 'all' && searching.value) return;
   searching.value = false;
   filterIconList();
 });
+
 onMounted(() => {
   isMobile.value = window.outerWidth <= 980;
+  // Check for existing search parameter in URL
+  const queryParams = new URLSearchParams(window.location.search);
+  const searchParam = queryParams.get('search');
+  if (searchParam !== null) {
+    search.value = searchParam;
+    searching.value = true;
```

Comment:
Not sure we should be setting searching.value = true here.... Does it get set to false when searching is complete?

Reply:
I removed it from here, didn't see the purpose either. 
I only see it used in
```
watch(selectedCategory, (newCategory) => {
  if (newCategory === 'all' && searching.value) return;
  searching.value = false;
  filterIconList();
});
```